### PR TITLE
wrong link is replaced by new link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report issue #5552.
i replaced the old link by new link

old link
https://go.microsoft.com/fwlink/?LinkId=99934

replaced link
https://docs.microsoft.com/en-in/windows-hardware/drivers/debugger/

added important note under Related topics

### CODE REVIEW

File: mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following items are required or recommended for creating the DaRT recovery i
  
 
 ## Related topics
-
+Note :: Debugging tools in not available for seperate download,Instead you must download Windows Driver Kit (WDK)
```
Comment (illfated) [inline mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]: I suggest moving this line up to before the section heading "Related topics"
```suggestion
```

File: mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ The following items are required or recommended for creating the DaRT recovery i
```
Comment (illfated) [inline mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]:
```suggestion
> [!NOTE]
> Debugging tools are not available as a separate download, but you can download them as part of the Windows Driver Kit (WDK).
```

Comment (RAJU2529) [reply mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]: @illfated.  thanks sir for correcting sentence.

**Comment by RAJU2529** [comment]
@illfated. Do you require any changes from my side?

**Comment by illfated** [comment]
Well, the suggestions are meant to be an improvement to the current content, but we can wait to see if @jvsam or @JohanFreelancer9 approve of them before any action is required.

File: mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md (hunk 2 above)
Comment (illfated) [reply mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]: @RAJU2529 : it looks like you have resolved this comment without merging the suggestion.
Would you like me to make a new suggestion to add the Note?

File: mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ The following items are required or recommended for creating the DaRT recovery i
...
 <td align="left"><p>Windows Debugging Tools for your platform</p></td>
-<td align="left"><p>Required when you run the <strong>Crash Analyzer</strong> to determine the cause of a computer failure. We recommend that you specify the path of the Windows Debugging Tools at the time that you create the DaRT recovery image. You can download the Windows Debugging Tools here: <a href="https://go.microsoft.com/fwlink/?LinkId=99934" data-raw-source="[Download and Install Debugging Tools for Windows](https://go.microsoft.com/fwlink/?LinkId=99934)">Download and Install Debugging Tools for Windows</a>.</p></td>
+<td align="left"><p>Required when you run the <strong>Crash Analyzer</strong> to determine the cause of a computer failure. We recommend that you specify the path of the Windows Debugging Tools at the time that you create the DaRT recovery image. You can download the Windows Debugging Tools here: <a href="https://docs.microsoft.com/en-in/windows-hardware/drivers/debugger/" data-raw-source="[Download and Install Debugging Tools for Windows](https://docs.microsoft.com/en-in/windows-hardware/drivers/debugger/)">Download and Install Debugging Tools for Windows</a>.</p></td>
```
Comment (JohanFreelancer9) [inline mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]:
```suggestion
<td align="left"><p>Required when you run the <strong>Crash Analyzer</strong> to determine the cause of a computer failure. We recommend that you specify the path of the Windows Debugging Tools at the time that you create the DaRT recovery image. You can download the Windows Debugging Tools here: <a href="https://docs.microsoft.com/windows-hardware/drivers/debugger/" data-raw-source="[Download and Install Debugging Tools for Windows](https://docs.microsoft.com/windows-hardware/drivers/debugger/)">Download and Install Debugging Tools for Windows</a>.</p></td>
```

**Comment by RAJU2529** [comment]
@JohanFreelancer9. Sir do you require any changes from my side?

**Comment by illfated** [comment]
@RAJU2529 : When your PR receives a Copy Review from JohanFreelancer9 like the one above, it is recommended that you merge the suggested change into your PR to continue the process to get your PR approved and merged.

File: mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md (hunk 3 above)
Comment (illfated) [inline mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]: @RAJU2529 : Please do as @JohanFreelancer9 requested, remove the locale `/en-in/` from the link if you want this PR approved.
```suggestion
<td align="left"><p>Required when you run the <strong>Crash Analyzer</strong> to determine the cause of a computer failure. We recommend that you specify the path of the Windows Debugging Tools at the time that you create the DaRT recovery image. You can download the Windows Debugging Tools here: <a href="https://docs.microsoft.com/windows-hardware/drivers/debugger/" data-raw-source="[Download and Install Debugging Tools for Windows](https://docs.microsoft.com/windows-hardware/drivers/debugger/)">Download and Install Debugging Tools for Windows</a>.</p></td>
```

File: mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md [hunk 4]
```diff
@@ -62,7 +62,6 @@ The following items are required or recommended for creating the DaRT recovery i
```
Comment (illfated) [inline mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]:
```suggestion
> [!NOTE]
> Debugging tools are not available as a separate download, but you can download them as part of the Windows Driver Kit (WDK).


```
Please also add this Note if you intend to do as you wrote in your PR opening comment, "added important note".

**Comment by RAJU2529** [comment]
@illfated. Now what should I do from my side sir

**Comment by illfated** [comment]
If you would be so kind, please use the Submit / Merge function within the suggested comments.

**Comment by RAJU2529** [comment]
@illfated. That I don't know, how to do, please provide steps to do with pictures

**Comment by illfated** [comment]
Very well, fair enough. First of all, there is a help page about this topic here:
- https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

1. Navigate to the first suggested change you'd like to apply.
     - To apply the change in its own commit, click **[Commit suggestion]**.

image
***
2. In the commit message field, type a short, meaningful commit message that describes the change you made to the file or files.

image
***
3. Click **[Commit changes]**.

image
***

**Comment by illfated** [comment]
I hope my comment above is clear and accurate enough for you to see what the procedure needs to be. Notice that the text in your own [Commit suggestion] dialog boxes will be slightly different, but only for the text content itself (for example, **[Update planning-to-create-the-dart-10-recovery-image.md]** automatically filled in when you open the [Commit suggestion] dialog. You can either choose to keep the text as it is, modify it if you find it reasonable to shorten the message, or simply add a secondary description underneath in the "**Add an optional extended description...**" text field.

**Comment by RAJU2529** [comment]
@ilfated. Now I am in college. I will do afternoon once I reached to home. Thanking you

File: mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md (hunk 4 above)
Comment (Dansimp) [reply mdop/dart-v10/planning-to-create-the-dart-10-recovery-image.md]: I removed en-in from the URL

**Pull request merged**